.github/workflows/code-nightly.yml

```diff
@@ -50,6 +50,7 @@ jobs:
         env:
           SLACK_WEBHOOK: ${{ secrets.IDE_SLACK_WEBHOOK }}
           SLACK_COLOR: ${{ job.status }}
+          SLACK_FOOTER: "<https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|See workflow logs here>"
```

Comment:
nit: would it make sense to have "View Error Details" or "Error logs" here for simplicity? Borrowed both from our GCP integration in Slack.

Reply:
True, maybe "Show logs" or "Workflow logs"?

Reply:
Updated